src/test/java/io/nats/client/impl/NatsMessageTests.java

```diff
@@ -123,15 +122,13 @@ public void testBigProtocolLineWithBody() {
  
     @Test
     public void testJSMetaData() {
-        byte[] body = new byte[10];
-        String subject = "subj";
         String replyTo = "$JS.ACK.test-stream.test-consumer.1.2.3.1605139610113260000";
 
-        Message msg = new NatsMessage(subject, replyTo, body, false);
+        Message msg = new NatsMessage.IncomingBuilder("sid", "subj", replyTo, 0).getMessage();

```

Comment:
The only way to make a JetStream message is to use the factory.

Reply:
Not for this PR, but it might be good at some point to open up creation of JS messages.   Users testing their application code could benefit from an easy way to artificially create JS messages.